velodrome/config.yaml

```diff
@@ -115,6 +115,28 @@ projects:
             static_configs:
               - targets:
                 - prow.k8s.io
+          - job_name: 'blackbox'
+            methids_path: /probe
+            params:
+              module: [http_2xx]
+            static_configs:
+              - targets:
+                - http://prow.k8s.io
+                - http://prow.k8s.io/hook
```

Comment:
This won't return 2XX, but 405. Maybe make a change to prow such that GET requests to this location return 200?

Reply:
I'm not sure how to do a POST without creating an entirely new scraping target. Even then, I'd need to change prow to ignore the github headers.

I think you're right that making prow return a 200 for GET is a more reasonable change.